src/components/ResultedMovies/MovieCard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function MovieCard(movie) {
```

Comment:
the argument should either be props or a destructuring object like `{singleMovie}` or `{movie}` if you change the prop name